src/arrays.js

```diff
@@ -1,61 +1,93 @@
 const getNthElement = (index, array) => {
-  // your code here
+  if (index > array.length - 1) {
+    index -= array.length;
+  }
+  return array[index];
 };
 
-const arrayToCSVString = array => {
-  // your code here
-};
+const arrayToCSVString = array => array.toString();
 
-const csvStringToArray = string => {
-  // your code here
-};
+const csvStringToArray = string => string.split(',');
 
 const addToArray = (element, array) => {
-  // your code here
+  array.push(element);
 };
 
-const addToArray2 = (element, array) => {
-  // your code here
-};
+const addToArray2 = (element, array) => array.concat(element);
 
-const removeNthElement = (index, array) => {
-  // your code here
-};
+const removeNthElement = (index, array) => array.splice(index, 1);
 
 const numbersToStrings = numbers => {
-  // your code here
+  const arr = numbers.map(item => {
+    return item.toString();
+  });
+  return arr;
```

Comment:
You dont need to create a variable to do this, just return the value directly:
```suggestion
  return numbers.map(item => {
    return item.toString();
  });
```